### ISSUE TITLE
Remove tasks from project

### ISSUE THREAD

**Issue by mtczerwinski** [kanet77/togglv8#8]
I have a problem with deleting tasks from a given project, especially here https://github.com/kanet77/togglv8/blob/master/lib/togglv8/tasks.rb#L36-L50 is something which has not got any sense. In a first function, you pass a `task_id` and call `join` on it and in a second function you do exactly the same thing but you change name of variable for `task_ids` then gem received 404 error from Toggl API and there is raised
```
NoMemoryError at /toggl/sync
Too deeply nested.
```
Here is a log:
```
D, [2015-12-09T12:34:46.803273 #96339] DEBUG -- : [httplog] Connecting: www.toggl.com:443
D, [2015-12-09T12:34:47.332799 #96339] DEBUG -- : [httplog] Sending: DELETE http://www.toggl.com:443/api/v8/tasks/12636535
D, [2015-12-09T12:34:47.333476 #96339] DEBUG -- : [httplog] Data:
D, [2015-12-09T12:34:47.507476 #96339] DEBUG -- : [httplog] Status: 404
D, [2015-12-09T12:34:47.508035 #96339] DEBUG -- : [httplog] Benchmark: 0.1732813100097701 seconds
D, [2015-12-09T12:34:47.508595 #96339] DEBUG -- : [httplog] Response:
```

**Comment by kanet77** [comment]
You're right that `delete_task` shouldn't include `.join(',')`. Good catch. I'll remove it.

`delete_task` accepts a single `task_id` (either an integer or string). `delete_tasks` accepts an array of `task_ids` (either integers or strings).

While I included tasks.rb for completeness, I don't use tasks. I see this is the second defect in tasks that you've found. Thanks for reporting them.

**Comment by kanet77** [comment]
Fixed this issue.

Now running tests for pro account features in Travis CI. Due to this, test coverage has increased to 97%.

**Comment by mtczerwinski** [comment]
Are you going to release a new version of gem by rubygems.org?

**Comment by kanet77** [comment]
Released v1.0.2

On Sat, Dec 12, 2015 at 3:38 PM Mateusz Czerwiński
notifications-at-github.com |github| <hr1ir7knet@sneakemail.com> wrote:

> Are you going to release a new version of gem by rubygems.org?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kanet77/togglv8/issues/8#issuecomment-164195616>.
>
-- 
Tom

--
@tomkane | 512.827.TOMK | ****************** |
http://www.linkedin.com/in/tominaustin
